apollo-runtime/src/main/java/com/apollographql/android/impl/ResponseBodyConverter.java

```diff
@@ -53,6 +59,7 @@
         }
       }
       jsonReader.endObject();
+      this.cache.cacheStore().merge(responseNormalizer.records());
```

Comment:
Good call. The normalizer should be a single use object, so changed it to have `RealApolloCall` send it `ResponseBodyConverter` and then save the records.

Also, good point about `responseNormalizer` crashing. In theory, if it's built right, and we get  a well-structured graphql response, it won't throw an error. On a malformed response, it may throw something. Not sure who would error first -- the generated response reading, or the normalizer in that case.